doc/dev/GETTING_STARTED.md

```diff
@@ -24,19 +24,25 @@ then install the required dependencies using the terminal.
 #### Debian/Ubuntu/Mint
 
 ```
+sudo apt update
+sudo apt upgrade
 sudo apt install build-essential git git-lfs cmake libvtk9-dev
 ```
 
 #### Fedora/Centos/RedHat
 
 ```
+sudo yum update
 sudo yum install make automake gcc gcc-c++ kernel-devel git git-lfs cmake vtk
+
 ```
 
 #### Arch Linux
 
 ```
-sudo pacman -S base-devel git git-lfs cmake vtk
+sudo pacman -Syu
+sudo pacman -S base-devel git git-lfs cmake vtk nlohmann-json hdf5 netcdf fmt verdict
```

Comment:
`netcdf` already depends on `hdf5` so you don't have to specify it explicitly, also `ospray` is need for the raytracing backend.

```suggestion
sudo pacman -S base-devel git git-lfs cmake vtk nlohmann-json netcdf fmt verdict ospray
```

Reply:
ospray is an optional dependecy, the guide doesnt explain how to get optional dependencies up

Reply:
same for `netcdf` and `hdf5` even if the plugin is enable by default.

Reply:
Let me spin up an Arch linux VM and check if I can compile without ospray and hdf5 specified

Reply:
@Meakk true, so should the guide focus on the default CMake configuration of F3D instead ?

Reply:
@Meakk ping ?

Reply:
I guess so since it's just a "getting started". If more advanced f3d configuration is needed, then we can assist on discord.

Reply:
So I guess the new suggestion omitting `ospray`? Basically your change but with `hdf5` removed.

```suggestion
sudo pacman -S base-devel git git-lfs cmake vtk nlohmann-json netcdf fmt verdict
```

Reply:
isnt hdf5 needed for exodus stuff ?

Reply:
@kidharb you basically need the change that meak suggest above. hdf5 is needed but it is a dependency of netcdf as @Meakk pointed out earlier.